AsyncDisplayKit/Details/ASRangeController.mm

```diff
@@ -320,6 +336,11 @@ - (void)_updateVisibleNodeIndexPaths
   _rangeIsValid = YES;
   _queuedRangeUpdate = NO;
   
+  for (void (^completionBlock)(void) in _scheduledRangeUpdateCompletionBlocks) {
+    completionBlock();
```

Comment:
This does seem like a useful pattern, although it is an interesting question on whether we will keep it long-term after addressing the root need for this change. It could actually turn out to be something we invest in and use more.